Directory.Packages.props

```diff
@@ -36,56 +36,56 @@
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.7" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="9.0.7" />
+    <PackageVersion Include="Microsoft.Extensions.ObjectPool" Version="9.0.7" />
+    <PackageVersion Include="Microsoft.IdentityModel.JsonWebTokens" Version="8.13.0" />
     <PackageVersion Include="Microsoft.IO.RecyclableMemoryStream" Version="3.0.1" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.14.0" />
     <PackageVersion Include="Microsoft.Testing.Extensions.CodeCoverage" Version="17.14.2" />
-    <PackageVersion Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.21.2" />
+    <PackageVersion Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.22.1" />
     <PackageVersion Include="Nethermind.Crypto.Bls" Version="1.0.5" />
     <PackageVersion Include="Nethermind.Crypto.SecP256k1" Version="1.4.1" />
     <PackageVersion Include="Nethermind.Crypto.SecP256r1" Version="1.0.0-preview.6" />
-    <PackageVersion Include="Nethermind.DotNetty.Buffers" Version="1.0.1" />
-    <PackageVersion Include="Nethermind.DotNetty.Handlers" Version="1.0.1" />
-    <PackageVersion Include="Nethermind.DotNetty.Transport" Version="1.0.1" />
+    <PackageVersion Include="Nethermind.DotNetty.Buffers" Version="1.0.2.76" />
+    <PackageVersion Include="Nethermind.DotNetty.Handlers" Version="1.0.2.76" />
+    <PackageVersion Include="Nethermind.DotNetty.Transport" Version="1.0.2.76" />
     <PackageVersion Include="Nethermind.GmpBindings" Version="1.0.2" />
     <PackageVersion Include="Nethermind.Libp2p" Version="1.0.0-preview.45" />
     <PackageVersion Include="Nethermind.Libp2p.Protocols.PubsubPeerDiscovery" Version="1.0.0-preview.45" />
     <PackageVersion Include="Nethermind.MclBindings" Version="1.0.0" />
     <PackageVersion Include="Nethermind.Numerics.Int256" Version="1.3.6" />
     <PackageVersion Include="Nito.Collections.Deque" Version="1.2.1" />
     <PackageVersion Include="NJsonSchema" Version="11.3.2" />
-    <PackageVersion Include="NLog" Version="5.5.0" />
-    <PackageVersion Include="NLog.Targets.Seq" Version="4.0.1" />
+    <PackageVersion Include="NLog" Version="6.0.2" />
+    <PackageVersion Include="NLog.Targets.Seq" Version="4.0.2" />
     <PackageVersion Include="NonBlocking" Version="2.1.2" />
     <PackageVersion Include="NSubstitute" Version="5.3.0" />
     <PackageVersion Include="NUnit" Version="4.3.2" />
     <PackageVersion Include="NUnit.Analyzers" Version="4.9.2" />
     <PackageVersion Include="NUnit3TestAdapter" Version="5.0.0" />
     <PackageVersion Include="Open.NAT.Core" Version="2.1.0.5" />
-    <PackageVersion Include="PierTwo.Lantern.Discv5.WireProtocol" Version="1.0.0-preview.4" />
-    <PackageVersion Include="Polly" Version="8.6.1" />
+    <PackageVersion Include="PierTwo.Lantern.Discv5.WireProtocol" Version="1.0.0-preview.5" />
```

Comment:
This version is defective, please, don't update until we fix it in preview.6 or so